plugins/passwordagentauthplugin/passwordagentplugin.cpp

```diff
@@ -586,6 +615,30 @@ void PasswordAgentPlugin::startDeviceLockAuthentication(
             promptText.message(),
             { { QStringLiteral("authenticatingPid"), QVariant::fromValue(callerPid) } },
             methods);
+
+    m_nemoTimers.insert(QPair<uint, quint64>(callerPid, requestId), timeout);
+}
+
+void PasswordAgentPlugin::cancelAuthentication(
+        uint callerPid,
+        qint64 requestId)
+{
+    QHash<QString, PolkitResponse *>::Iterator it = m_polkitResponses.begin();
+    while (it != m_polkitResponses.end()) {
+        if (it.value()->callerPid == callerPid
+            && it.value()->requestId == requestId) {
```

Comment:
style nitpick: double indent condition continuation lines, to visually separate from the entailed code block

Reply:
Ok, done.